dataactvalidator/config/sqlrules/b20_object_class_program_activity.sql

```diff
@@ -37,6 +44,9 @@ WHERE NOT EXISTS (
                 OR COALESCE(af.program_activity_code, '') = ''
                 OR af.program_activity_code = '0000'
             )
+            AND (UPPER(COALESCE(af.program_activity_name, '')) = UPPER(COALESCE(op.program_activity_name, ''))
```

Comment:
program activity name is not indexed in our tables. We may want to change that considering we'll be running this rule on it. I think we can just tweak the migration we already have, no need for a new one, to also index `UPPER(program_activity_name)`